### ISSUE TITLE
Add New Categories & Sites to Homescreen as Experiment (code)

### ISSUE THREAD

**Issue by athomasmoz** [mozilla-mobile/firefox-tv#2195]
### Why/User Benefit/User Problem

As a Firefox for Fire TV user, I want more media sites on the home screen, so I know where to find content & can easily navigate to it

### Acceptance Criteria (how do I know when I’m done?)

- [x] New MVP channels have been added to EN_US-only home screen (**and not other locales**)
- [x] Channel matches the current design of pinned sites
- [x] Telemetry added for each tile within the category
      * Event for tile click that includes tile name - same as pinned tiles telemetry (See #2262 for requirements)
- [x] This will be put behind an experiment
- [x] Add 10 tiles per channel, from the sites spreadsheet - only add APPROVED tiles
- [x] For categories that don't have 10 tiles, **add duplicates of approved channels** so we have an idea of what they look like.
- [ ] **On 5/30** after #2322 is finished match the sites/categories in the app to the final list. (Remove duplicates.)
- [ ] After final sites added, take screenshots for SUMO article and update them: https://docs.google.com/document/d/1pXGanTld9lyIgRVASSOi4U466CPehhlrhlmDnMUOlsg/edit
- [x] Testing for if all on/off for experiment, or EN-US - should not break

### Notes

* Final list of categories (including titles) & sites will come out of #2322 
* Logos for the tiles (provided here: https://github.com/mozilla-mobile/firefox-tv/issues/2209#issuecomment-492082424 )
* This does not include removing tiles (that is #2326 )
* The channels will be hidden behind a Fretboard experiment in the second week of the sprint (#2321 )

**Comment by liuche** [comment]
Content team is looking at this, but actually, I don't think these strings need to be localized because this is en-US only, right?
@Delphine what do you think, we doing an MVP of this "tv channel guide" but we won't be shipping these (now) for non-en-US because we don't have a curated list of these sites. To keep it simple though, we'll land these to be localized, but won't be showing them to everyone yet.

**Comment by Delphine** [comment]
If these strings are going to end up being localized in the end, then I think this works. thanks!

**Comment by athomasmoz** [comment]
Suggested approach:
* News & Politics - Confirmed. Has 6 sites
* Sports - Test Post-MVP to see if we can get a few more sites
* Music - Test Post-MVP to see if we can get a few more sites
* Features Video - Cut. No Post-MVP site list to test.
* Culture - Cut. No Post-MVP site list to test.
* Food - Test Post-MVP sites here to see if we can add this channel

**Comment by Baron-Severin** [comment]
Major engineering work is done (unless we're blocking on https://github.com/mozilla-mobile/firefox-tv/issues/2289#issuecomment-497517136).  We'll be able to finish up and land once we have:
- The final list of websites
- Tile image assets for each website
- String titles for each website confirmed (most were self evident, but some are iffy [e.g., under music, 'NPR' or 'NPR Stations'?])
- Strings for channel titles confirmed (I took these from the Excel tabs)

**Comment by liuche** [comment]
@Baron-Severin one thing I remembered was that **Branch A** should be the control, not branch B, according to our Telemetry docs: https://github.com/mozilla-mobile/firefox-tv/blob/master/docs/telemetry.md

That's probably a mistake I made in speccing it out, is that something you can change in this bug?

**Comment by Baron-Severin** [comment]
> @Baron-Severin one thing I remembered was that **Branch A** should be the control, not branch B, according to our Telemetry docs: https://github.com/mozilla-mobile/firefox-tv/blob/master/docs/telemetry.md
> 
> That's probably a mistake I made in speccing it out, is that something that you've already put into this, or the killswitch bug? If it's #2320 then I'll take care of it.

Thank you, good catch.  I can fix that as part of this bug.
